Sources/Danger/GitHubDSL.swift

```diff
@@ -318,7 +318,7 @@ extension GitHub {
         public let commitId: String?
 
         /// The id for the review (if a review was made).
-        public let id: Int? // swiftlint:disable:this identifier_name
+        public let id: Int? 
```

Comment:
<!--
  0 failure: 
  1 warning:  Lines should not ...
  
  
  DangerID: danger-id-Danger;
  File: Sources/Danger/GitHubDSL.swift;
  Line: 321;
-->

- :warning: Lines should not have trailing whitespace. (`trailing_whitespace`)